README.md

```diff
@@ -45,37 +61,62 @@ You'll also need to install a couple things that involve Gits. To do this, you n
 2. Follow instructions to install the application properly.
 3. Run `haxelib git polymod https://github.com/larsiusprime/polymod.git` to install Polymod.
 4. Run `haxelib git discord_rpc https://github.com/Aidan63/linc_discord-rpc` to install Discord RPC.
+5. Optional: - Run `haxelib git flixel-addons https://github.com/HaxeFlixel/flixel-addons` to update Flixel-Addons. This fixes the transition bug for zoomed out stage cameras.
 
 You should have everything ready for compiling the game! Follow the guide below to continue!
 
-At the moment, you can optionally fix the transition bug in songs with zoomed out cameras.
-- Run `haxelib git flixel-addons https://github.com/HaxeFlixel/flixel-addons` in the terminal/command-prompt.
-
-### Ignored files
+# Adding `APIStuff.hx` into `/source`
 
-I gitignore the API keys for the game, so that no one can nab them and post fake highscores on the leaderboards. But because of that the game
-doesn't compile without it.
+![APIStuff.hx_picture](https://user-images.githubusercontent.com/72235309/112080256-00b24c80-8b58-11eb-9499-524bf45a00eb.png)
 
-Just make a file in `/source` and call it `APIStuff.hx`, and copy paste this into it
+The creator gitignored the API keys of the game so no one could post fake high scores onto the leaderboards in Newgrounds. Unfortunatly, because this game requires the `API` and `EncKey` values to compile, you will need to add a file called `APIStuff.hx` into `/source`. How do I do that?

```

Comment:
`The creator ...` sounds a bit off to me; I think it should instead say `The API keys have been gitignored so ...` That way, it sounds more natural.

Also, you spelt `unfortunatly` wrong. Mistakes happen. To err is human, after all.